packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseServerObject-test.js

```diff
@@ -0,0 +1,79 @@
+const { Fury } = require('fury');
+const { expect } = require('../../chai');
+
+const parse = require('../../../../lib/parser/oas/parseServerObject');
+const Context = require('../../../../lib/context');
+
+const { minim: namespace } = new Fury();
+
+describe('#parseHostsObject', () => {
```

Comment:
```suggestion
describe('#parseServerObject', () => {
```

I think this has been renamed.

Reply:
returned to old denomination

Reply:
I think this is still valid, the name here should match the function we are testing, which is `parseServerObject`, from:

```js
const parse = require('../../../../lib/parser/oas/parseServerObject');
```

Reply:
yes, changed, sorry